ui/imports/shared/controls/StatusSyncCodeScan.qml

```diff
@@ -77,9 +77,9 @@ Column {
     Component {
         id: btnComponent
 
-        Rectangle {
+        ShapeRectangle {
```

Comment:
Forgot to mention that; using the `ShapeRectangle` now instead of a plain one (as in Figma)